packages/svelte/src/internal/client/dom/elements/attachments.js

```diff
@@ -0,0 +1,11 @@
+import { effect } from '../../reactivity/effects.js';
+
+/**
+ * @param {Element} node
+ * @param {() => (node: Element) => void} get_fn
+ */
+export function attach(node, get_fn) {
+	effect(() => {
+		return get_fn()(node);
```

Comment:
```suggestion
		return (get_fn())?.(node);
```

would resolve https://github.com/sveltejs/svelte/pull/15000#issuecomment-2591570962

[repl click on button for error](https://svelte.dev/playground/hello-world?version=pr-15000#H4sIAAAAAAAAE22RwW6DMAyGX8WNJgESYnda2u0y9QF2GztQYtRswYkSQ1ch3n0KlJZJuyV_Ptv_7wyCqhZFLo6otYGLcVpCjFIxykSkolEavcg_BsFXG7ggiHSperU28z1qDtqp8vifXhtiJPYiFztfO2V5X1LJqrXGMbCy9gqNMy1E0zn78tG2pIBoZLhVQwFPnivGOJqsbqJkuyAXxed3YzQr-8DYdZis-vAMvFFAJDrVo4zXlYcFgRw6ktgoQrl0aDqqWRlamPhmK4EhPJfskDtHEJORmECxX_SSa0P-Ph6KOfDEpTDc441znFWrW0Um0bMz123JMzBO4FjS7vmxTdqdOmZDMLxUzFV9fuQ9ZHezIxiqtaq_iyGeTP7d3GZ1Hac_OpoeHbQYhs0D9iIVjD8s8rDg8XP8BfjVJiNCAgAA)

Reply:
thanks, was meaning to get round to this!